src/Microsoft.AspNetCore.Http/Features/FormOptions.cs

```diff
@@ -8,6 +8,8 @@ namespace Microsoft.AspNetCore.Http.Features
 {
     public class FormOptions
     {
+        public static readonly FormOptions Default = new FormOptions();
```

Comment:
We can't make a mutable instance a public static

Reply:
Good point